### PR TITLE
Test api

### PR DESCRIPTION
**Pull request by anthonysim**

### CODE REVIEW

**Comment by anthonysim** [comment]
Screen Shot 2021-05-01 at 4 25 11 PM

File: __testing__/apiEnrolled.test.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+it('Should POST enrolled number to database', async (done) => {
+  const postedEnrolledNum = '18789';
+
+  const res = await request.post('/postEnrolled')
+    .send({
+      id: '1',
+      enrolled: postedEnrolledNum,
+    });
+
+  expect(res.status).toBe(200);
```
Comment (kmgehrma) [inline __testing__/apiEnrolled.test.js:25]: You are testing all the successes (which is good), but do you also want to test failures / make sure the proper error status is returned when it's expected?

File: server/index.js [hunk 2]
```diff
@@ -34,6 +34,15 @@ app.get('/*', (req, res) => {
   res.sendFile(path.resolve(__dirname, '../public/index.html'));
 });
 
+// server connection
+const PORT = process.env.PORT || 3001;
+
+if (process.env.NODE_ENV !== 'test') {
```
Comment (kmgehrma) [inline server/index.js:40]: Nice use of environment variables; I appreciated this conversation on Slack / learning that NODE_ENV is set to 'test' during testing

**Review by kmgehrma — CHANGES_REQUESTED** [review]
All in all your tests look solid. You are testing all the success scenarios, but I might throw in an error scenario or two. Also, I am wondering where the file which contains the implementation of your CRUD routes is.

File: server/index.js (hunk 2 above)
Comment (anthonysim) [reply server/index.js:40]: > All in all your tests look solid. You are testing all the success scenarios, but I might throw in an error scenario or two. Also, I am wondering where the file which contains the implementation of your CRUD routes is.

Yup, that makes total sense! I'll pull this request into main, but I will definitely work on error scenarios and add them in.

**Pull request merged**